### PR TITLE
Remove DL's from portal message templates.

### PR DESCRIPTION
**Pull request by khink**
https://github.com/plone/Products.CMFPlone/issues/153

### CODE REVIEW

**Comment by jensens** [comment]
do we need this one?

**Comment by jensens** [comment]
@khink do we still need this one?

**Comment by khink** [comment]
I'd say yes

**Comment by jensens** [comment]
So can you please rebase with master, so we can merge it? :D

File: plone/app/contentrules/browser/templates/manage-elements.pt [hunk 1]
```diff
@@ -282,6 +288,19 @@
 
           </tal:noassignments>
           <tal:assignments condition="nocall:assignments">
+<<<<<<< HEAD
```
Comment (jensens) [inline plone/app/contentrules/browser/templates/manage-elements.pt]: looks like some merge conflicts are left

File: plone/app/contentrules/browser/templates/manage-elements.pt [hunk 2]
```diff
@@ -220,9 +220,15 @@
                            class="context"
                            />
                 </div>
+<<<<<<< HEAD
```
Comment (jensens) [inline plone/app/contentrules/browser/templates/manage-elements.pt]: here too

**Comment by jensens** [comment]
http://jenkins.plone.org/view/All/job/pull-request/25/ passes

**Pull request merged**